samples/azure/storage-persistence/ASP_2/StorageReader/AzureHelper.cs

```diff
@@ -31,45 +32,61 @@ public Task WriteOutData()
         var tableClient = storageAccount.CreateCloudBlobClient();
         var container = tableClient.GetContainerReference(containerName);
         Debug.WriteLine($"'{containerName}' container contents");
-        foreach (var blob in container.ListBlobs())
+        BlobContinuationToken token = null;
+        do
         {
-            var name = blob.Uri.AbsolutePath.Split('/').Last();
-            Debug.WriteLine($"  Blob:= {name}");
-            var blockBlobReference = container.GetBlockBlobReference(name);
-            var text = await blockBlobReference.DownloadTextAsync()
-                .ConfigureAwait(false);
-            Debug.WriteLine($"    {text}");
+            var segment = await container.ListBlobsSegmentedAsync(token);
```

Comment:
I think this requires `ConfigureAwait(false)` we apply that to the doco snippets

Reply:
addressed